yarn.lock

```diff
@@ -4394,7 +4419,7 @@ expect@^26.0.1:
     jest-message-util "^26.0.1"
     jest-regex-util "^26.0.0"
 
-express@^4.17.1:
+express@^4.16.3, express@^4.17.1:
   version "4.17.1"
   resolved "https://registry.yarnpkg.com/express/-/express-4.17.1.tgz#4491fc38605cf51f8629d39c2b5d026f98a4c134"
   integrity sha512-mHJ9O79RqluphRrcw2X/GTh3k9tVv8YcoyY4Kkh4WDMUYKRZUq0h1o0w2rrrxBqM7VoeUVqgb27xlEMXTnYt4g==
```

Comment:
It's a little weird that we're getting 2 versions of this now, right?

Reply:
I think this line just lists out all dependency requirements, in which `express@^4.16.3` is required by webpack-bundle-analyzer:
```
express@^4.16.3, express@^4.17.1:
```
The next line shows the actual version installed (which satisfied both requirements):
``` 
  version "4.17.1"
```

Reply:
Ah 👍